install/pre-install-payload/scripts/reqs-deploy.sh

```diff
@@ -195,8 +195,11 @@ function configure_nydus_snapshotter_for_containerd() {
 	address = "/run/containerd-nydus/containerd-nydus-grpc.sock"
 EOF
 		if grep -q "^imports = " "$tmp_containerd_config"; then
```

Comment:
Small improvement (matter of personal taste):

```bash
local imports
imports=$(grep "^imports = " "$tmp_containerd_config")
if [[ -n "${imports}" && ! "$imports" =~ "\"${containerd_imports_path}/nydus-snapshotter.toml\"" ]]; then
...
...
fi
```

Reply:
I'd like to avoid changing multiple things (this is preexisting code, only changing the variable) in one commit so let's leave it as is but feel free to send mass-style-fix afterwards ;-)